bedrock/firefox/views.py

```diff
@@ -526,3 +527,51 @@ def ios_testflight(request):
     return l10n_utils.render(request,
                              'firefox/testflight.html',
                              {'newsletter_form': newsletter_form})
+
+
+class FirefoxProductDesktopView(BlogPostsView):
+    blog_posts_limit = 3
+    blog_posts_template_variable = 'articles'
+    blog_slugs = 'firefox'
+
+    def get_template_names(request):
+        locale = l10n_utils.get_locale(request)
+
+        if locale == 'en-US':
+            template_name = 'firefox/products/desktop.html'
+        else:
+            template_name = 'firefox/desktop/index.html'
+
+        return [template_name]
+
+
+class FirefoxProductAndroidView(BlogPostsView):
+    blog_posts_limit = 3
+    blog_posts_template_variable = 'articles'
+    blog_slugs = 'firefox'
```

Comment:
> I think we'll want to recycle jpetto's logic on filtering here, to avoid repeating posts that are tagged both "mobile" and "featured."

I don't think that should be the case here - it won't pull in the same post twice. I think Jon's case was different.